app/permissions/PermissionsActionCheck.scala

```diff
@@ -27,6 +27,56 @@ trait PermissionActionFilter extends ActionFilter[UserRequest] with Logging {
     }
   }
 }
+trait ModifyCollectionsPermissionsCheck extends Logging { self: BaseFaciaController =>
+
+  def acl: Acl
+  def configAgent: ConfigAgent
+
+  private def testAccess(email: String, priorities: Set[PermissionsPriority], isLaunch: Boolean) = {
+
+    if (priorities.isEmpty) AccessGranted
+    else if (isLaunch)
+      acl.testUserGroupsAndCollections(
+        Permissions.LaunchEditorialFrontsAlert, Permissions.LaunchCommercialFrontsAlert,
+        Permissions.FrontsAccess, "facia-tool-allow-access-for-all")(email, priorities)
+    else
+      acl.testUserGroupsAndCollections(
+        Permissions.EditEditorialFrontsAlert, Permissions.LaunchCommercialFrontsAlert,
+        Permissions.FrontsAccess, "facia-tool-allow-access-for-all")(email, priorities)
```

Comment:
We need to add the correct switch name here as well, don't we @susiecoleman ? It's going to be messy because we are going to have to figure out the correct switch name from the priority names.

Reply:
I think something might have got lost in a merge? As this is right on my machine. These should be `"facia-tool-allow-launch-editorial-fronts-for-all"` and `"facia-tool-allow-edit-editorial-fronts-for-all"` respectively. I'll push up the changes